<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chipTanDialog</class>
 <widget class="QWidget" name="chipTanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Order confirmation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="reduceButton">
         <property name="toolTip">
          <string>Reduce optical tan area in size</string>
         </property>
         <property name="text">
          <string>Reduce</string>
         </property>
         <property name="icon">
          <iconset theme="zoom-out">
           <normaloff/>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="enlargeButton">
         <property name="toolTip">
          <string>Enlarge optical tan field</string>
         </property>
         <property name="text">
          <string>Enlarge</string>
         </property>
         <property name="icon">
          <iconset theme="zoom-in">
           <normaloff/>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="decelerateButton">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>Decelerate</string>
         </property>
         <property name="icon">
          <iconset theme="media-seek-backward">
           <normaloff/>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="accelerateButton">
         <property name="text">
          <string>Accelerate</string>
         </property>
         <property name="icon">
          <iconset theme="media-seek-forward">
           <normaloff/>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QTextBrowser" name="infoText">
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="acceptRichText">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDeclarativeView" name="declarativeView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>360</width>
         <height>300</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>To confirm this order enter the tan displayed by your generator</string>
       </property>
       <property name="buddy">
        <cstring>tanInput</cstring>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KLineEdit" name="tanInput"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="KDialogButtonBox" name="dialogButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDeclarativeView</class>
   <extends>QGraphicsView</extends>
   <header>QtDeclarative/QDeclarativeView</header>
  </customwidget>
  <customwidget>
   <class>KDialogButtonBox</class>
   <extends>QDialogButtonBox</extends>
   <header>kdialogbuttonbox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tanInput</tabstop>
  <tabstop>infoText</tabstop>
  <tabstop>declarativeView</tabstop>
  <tabstop>dialogButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
